pages/research/department/[id].tsx

```diff
@@ -155,6 +175,81 @@ const Page = ({
             </Card>
           </div>
         </Section>
+        <Section>
+          <h2>Travels</h2>
+          <Paragraph>
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Amet
+            molestiae, sequi animi est dolor nihil qui id, aperiam assumenda
+            suscipit officia, veniam tenetur veritatis saepe! Recusandae animi
+            incidunt fuga perferendis!
+          </Paragraph>
+          <TravelsOfDepartmentPrismMap
+            topology={getCountries()}
+            topologyObject={"ne_admin_0_countries"}
+            projection={geoBertin1953()}
+            width={10}
+            length={10}
+            extrudeGeometryOptions={{
+              depth: 0.01,
+              bevelSize: 0.005,
+              bevelThickness: 0.005,
+              bevelSegments: 12,
+            }}
+            btorsByCountryByDepartment={btorsByCountryByDepartment}
+            department={department}
```

Comment:
I get an typescript error for this line. One possible quick fix (even though certainly not the best one would be to include the department number in the select of the `getDepartment` query,

Reply:
I saw this error too and will try to fix it.